### PR TITLE
bisonw: brand wallet

### PR DESCRIPTION
**Pull request by buck54321**
DCRDEX Client -> Bison Wallet

Application data directories and configuration file names are unchanged. An upgrade that moves those directories is feasible, but messy and pointless.
image

### CODE REVIEW

File: client/webserver/site/src/css/main.scss [hunk 1]
```diff
@@ -37,12 +37,12 @@ section {
...
-  background-size: cover;
-  cursor: pointer;
-  background-image: var(--logo-path);
-}
+// .logo {
+//   display: block;
+//   background-size: cover;
+//   cursor: pointer;
+//   background-image: var(--logo-path);
+// }
```
Comment (ukane-philemon) [inline client/webserver/site/src/css/main.scss:46]: intend to remove?

File: docs/wiki/Client-control-API.mediawiki [hunk 2]
```diff
@@ -66,7 +66,7 @@ The response will contain a help message based on the arguments.
 
 ===Version===
 
-Print the DEX client rpcserver version. Accepts no arguments
+Print the Bison Wallet rpcserver version. Accepts no arguments
```
Comment (ukane-philemon) [inline docs/wiki/Client-control-API.mediawiki:69]: In some places we no longer refer to `the Bison Wallet` but `Bison Wallet`, intentional? IF just `Bison Wallet` sounds better we should have that here and every other place below where we do `The Bison Wallet` I think.

**Review by ukane-philemon — CHANGES_REQUESTED** [review]
Nice job buck! mostly nits

Quick question: Do we intent to leave the `client` directory unchanged?

**Comment by exitus1** [comment]
Looks really good. 

If I could suggest something - it would be nice if the ID and the Amount were switched. It felt a bit odd during testing to have to pan my eyes to connect the Amount with the Type of transaction.

**Comment by buck54321** [comment]
> Nice job buck! mostly nits
> 
> Quick question: Do we intent to leave the `client` directory unchanged?

Yes

**Review by JoeGruffins — APPROVED** [review]
A couple dexc in error logs:
```
client/core/trade.go:2454:                      "to your wallet in the case dexc no longer functions. They should "+
client/core/trade.go:2455:                      "NOT be used if dexc is operable. dexc will refund failed "+
```

In translations? Unsure if these are ignored files but don't think they are:
```
client/webserver/locales/pl-pl.go:339:  "save_orders_to_file_msg":     {T: "Opcjonalne: czy zapisywać usunięte zamówienia w pliku CSV w folderze danych dexc. Domyślnie: nie."},
client/webserver/locales/pl-pl.go:340:  "save_matches_to_file_msg":    {T: "Opcjonalne: czy zapisywać usunięte spasowania w pliku CSV w folderze danych dexc. Domyślnie: nie."},
client/webserver/jsintl.go:736: connectionFailedID:               {T: "فشل الاتصال بخادم dex. يمكنك إغلاق dexc والمحاولة مرة أخرى لاحقًا أو انتظا�
```

File: docs/wiki/Client-control-API.mediawiki (hunk 2 above)
Comment (buck54321) [reply docs/wiki/Client-control-API.mediawiki:69]: The name is certainly **Bison Wallet** and not **The Bison Wallet**, but most of these are OK. In the statement `the Bison Wallet rpcserver version` or `the Bison Wallet password`, **the** points to **version** or **password**, and **Bison Wallet** just specifies **which** password or version.

**Pull request merged**